docs/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu  
+
+    attr_reader :order
+
+    def initialize
+       @list = { 'Latte' => 2.90, 'Cappuccino' => 2.80, 'Cake' => 3.50, 'Sandwich' => 4.00 }    
```

Comment:
Might be worth setting this to a constant variable as it is probably rarely that the menu will change.